fdbclient/NativeAPI.actor.cpp

```diff
@@ -2692,8 +2708,22 @@ void debugAddTags(Transaction *tr) {
 
 }
 
+SpanID generateSpanID(int transactionTracingEnabled) {
+	UID uid = deterministicRandom()->randomUniqueID();
+	if (transactionTracingEnabled > 0) {
```

Comment:
Nit: you could call `randomInt64` instead of `randomUniqueID`. That way you will only need to generate one 8 byte int if tracing is disabled.